ppss.yaml

```diff
@@ -13,15 +13,24 @@ predictoor_ss:
   bot_only:
     s_until_epoch_end: 60 # in s. Start predicting if there's > this time left
     stake_amount: 1 # stake this amount with each prediction. In OCEAN
+  is_two_sided: False
   approach3:
-  aimodel_ss:
+  regressionmodel_ss:
     input_feeds:
       - binance BTC/USDT c 5m
 #      - binance BTC/USDT ETH/USDT BNB/USDT XRP/USDT ADA/USDT DOGE/USDT SOL/USDT LTC/USDT TRX/USDT DOT/USDT ohlcv 5m
 #    - kraken BTC/USDT 5m
     max_n_train: 5000 # no. epochs to train model on
-    autoregressive_n : 10 # no. epochs that model looks back, to predict next
+    autoregressive_n: 10 # no. epochs that model looks back, to predict next
     approach: LIN
+  classifiermodel_ss:
+    input_feeds:
+      - binance BTC/USDT c 1h
+    #      - binance BTC/USDT ETH/USDT BNB/USDT XRP/USDT ADA/USDT DOGE/USDT SOL/USDT LTC/USDT TRX/USDT DOT/USDT ohlcv 1h
+    #    - kraken BTC/USDT 1h
+    max_n_train: 5000 # no. epochs to train model on
+    autoregressive_n: 10 # no. epochs that model looks back, to predict next
```

Comment:
Many classifiermodel_ss attributes are identical to regressionmodel_ss. They're both referring to the same thing: how to construct the data. Currently, the only thing that should be different between classifier & regression is "approach" because they refer to different algorithms.

I recommend to structure things differently:
- have an aimodel_data_factory.py holding AimodelDataFactory. That class has methods create_xy_regression() and create_xy_classification(). The latter calls the former. The factory settings are in AimodelDataSS.
- have RegressionModelFactory like you're currently doing. The factory settings are in RegressionModelSS. Note that RegressionModelSS doesn't hold AimodelDataSS.
- similar to regression, have ClassificationModelFactory and ClassificationModelSS.

This is how the yaml would look:
```text
predictoor_ss:
  aimodel_data_ss:
     data:
        input_feeds:
        - binance BTC/USDT c 1h
#      - kraken BTC/USDT 1h
    max_n_train: 5000 # no. epochs to train model on
    autoregressive_n: 10 # no. epochs that model looks back, to predict next

  regressionmodel_ss:
    approach: LIN

  classifiermodel_ss:
     approach: LIN
```